# Protocols

<h2 style="margin: 0 !important;">1. Preparation of Broth and Culturing of Yeast</h2>

<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">Materials:</h3></li>
  <ul>
    <li>YPD Broth</li>
    <li>Distilled water</li>
    <li>100 mL flasks</li>
</ul>
    <li><h3 style="margin: 0 !important;">Procedure:</h3></li>
  <ul>
    <li>Suspend 50g of YPD Broth in 1 L of distilled water</li>
    <li>Autoclave for 15 minutes at 121 °C</li>
    <li>Inoculate yeast cultures sourced from yeast broths or plates in detergent-free flasks containing the prepared liquid medium</li>
    <li>Note that the medium should not be more than a fifth of the total flask volume</li>
    <li>Vortex the contents briefly to disperse cells.</li>
    <li>Grow the culture in a shaking incubator at 300 rpm in flasks.</li>
    </ul>
</ul>
<div style="color:gray;text-align: left;margin: 20px 0 20px 0;">Source:
https://www.sigmaaldrich.com/IN/en/technical-documents/protocol/microbiological-testing/pathogen-and-spoilage-testing/yeast-growth-protocols
</div>
<h2 style="margin: 0 !important;">2. Isolating yCD gene from yeast</h2>

<ul style="list-style-type:none;">
  <h3 style="margin: 0 !important;">a) Preparing yeast cells for lysis:</h3>
  <li><h3 style="margin: 0 !important;">Materials:</h3></li>
  <ul>
    <li>Yeast cells</li>
    <li>STES buffer: 0.2 M Tris-Cl (pH 7.6), 0.5 M NaCl, 0.01 M EDTA, 0.1% (w/v) SDS</li>
    <li>Microcentrifuge tube</li>
</ul>
    <li><h3 style="margin: 0 !important;">Procedure:</h3></li>
  <ul>
    <li>Transfer 1.5 mL from an overnight culture of yeast cells to a microcentrifuge tube.
</li>
    <li>Pellet the cells by centrifuging at maximum speed for 1 min at room temperature in a microcentrifuge</li>
    <li>Remove the culture medium by aspiration, and resuspend the pellet in 50 µL of STES buffer</li>
</ul>
<h3 style="margin: 0 !important;">b) Extracting genomic DNA from pelleted yeast cells:</h3>

<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">Materials:</h3></li>
  <ul>
    <li>Ethanol (70%, 95%)s</li>
    <li>Phenol:chloroform solution (1:1, v/v)</li>
    <li>STES buffer: 0.2 M Tris-Cl (pH 7.6), 0.5 M NaCl, 0.01 M EDTA, 0.1% (w/v) SDS</li>
    <li>TE buffer, 10× (pH 7.6): 100 mM Tris-Cl (desired pH) 10 mM EDTA (pH 8.0)
</li>
</ul>
    <li><h3 style="margin: 0 !important;">Procedure:</h3></li>
  <ul>
    <li>Add 50 µL of acid-washed glass beads to each microcentrifuge tube. Add 20 µL of TE (pH 7.6) to each tube
</li>
    <li>Add 60 µL of phenol:chloroform, cap the tubes tightly, and mix the organic and aqueous phases by vortexing for 1 min</li>
    <li>Centrifuge the tubes at maximum speed for 5 min at room temperature in a microcentrifuge.</li>
    <li>Transfer the upper aqueous phase to a fresh microcentrifuge tube. Collect the DNA by standard precipitation with ethanol for 5 min at 0˚C</li>
    <li>Recover the precipitate of nucleic acids by centrifugation at maximum speed for 10 min at 4˚C in a microcentrifuge</li>
    <li>Remove the supernatant by aspiration, and rinse the pellet with 100 µL of 70% ethanol. Centrifuge the tubes at maximum speed for 1 min at room temperature in a microcentrifuge.</li>
    <li>Remove the supernatant by aspiration, and allow the pellet to air-dry for 15 min. Redissolve the pellet in 40 µL of TE (pH 7.6)</li>
</ul>

<h3 style="margin: 0 !important;">c) Isolating yCD gene:</h3>

<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">Materials:</h3></li>
  <ul>
    <li>Yeast genomic DNA</li>
    <li>10X rCutSmartBuffer</li>
    <li>Ndel and Blpl restriction enzymes</li>
    <li>Nuclease-free water</li>
    
</ul>
    <li><h3 style="margin: 0 !important;">Procedure:</h3></li>
  <ul>
    <li>Mix 1 µg of the extracted yeast DNA, 5 µl of 1X rCutSmartBuffer, 1 1.0 µl (20 units) of Ndel, 1.0 µl (10 units) of blpl, and adjust the volume to 50 µl with nuclease-free water</li>
    <li>Incubate at 37°C for 1 hour</li>
</ul>
</ul>
<div style="color:gray;text-align: left;margin: 20px 0 20px 0;">Source: Cold Spring Harbor Protocols; doi: 10.1101/pdb.prot093542, 
https://nebcloner.neb.com/#!/protocol/re/double/NdeI,BlpI

</div>

<h3 style="margin: 0 !important;">3. Performing agarose gel electrophoresis identify the gene</h3>

<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">Materials and Equipment:</h3></li>
  <ul>
    <li>TAE 50x buffer: Tris-base - 242 g, 100% acetic acid - 57.1 ml, EDTA- 100 ml 0.5M sodium EDTA, dH2O up to 1L</li>
    <li>Agarose</li>
    <li>Ethidium bromide (stock concentration of 10 mg/mL)
</li>
    <li>Deionized water
</li>
    <li>PCR clean-up Gel column</li>
    <li>NT1, NT3, Elution Buffers</li>
    <li>Casting tray</li>
    <li>Well combs</li>
    <li>Voltage source</li>
    <li>Gel box</li>
    <li>UV light source</li>
    <li>Microwave</li>
    
</ul></ul>


<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">Procedure:</h3></li>
  <li><h3 style="margin: 0 !important;">a) Pouring a Standard 1% Agarose Gel:</h3></li>
  
<ul>
    <li>Measure 1 g of agarose</li>
    <li>Mix agarose powder with 100 mL 1xTAE in a microwavable flask.</li>
    <li>Microwave for 1-3 min until the agarose is completely dissolved.
</li>
    <li>Let the agarose solution cool down to about 50 °C in about 5 minutes
</li>
    <li>Add ethidium bromide (EtBr) to a final concentration of approximately 0.2-0.5 μg/mL</li>
    <li>Pour the agarose into a gel tray with the well comb in place.</li>
    <li>Place newly poured gel at 4 °C for 10-15 mins, or let sit at room temperature for 20-30 mins until it has completely solidified.</li>
    
    
</ul></ul>
    
 <ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">b) Loading Samples and Running an Agarose Gel:</h3></li>
  
  
<ul>   
<li>Add a loading buffer to each of the isolated yCD samples.
</li>
    <li>Once solidified, place the agarose gel into the gel box (electrophoresis unit).</li>
    <li>To make 1x TAE from 50X TAE stock, dilute 20ml of stock into 980 ml of deionized water.
</li>
    <li>Fill the gel box with 1xTAE (or TBE) until the gel is covered.
</li>
    <li>Carefully load a molecular weight ladder into the first lane of the gel.</li>
    <li>Carefully load your samples into the additional wells of the gel.
</li>
    <li>Run the gel at 80-150 V until the dye line is approximately 75-80% of the way down the gel</li>
    <li>Turn OFF power, disconnect the electrodes from the power source, and then carefully remove the gel from the gel box.</li>
    <li>Using any device that has UV light, visualize your DNA fragments as bands. </li>

</ul></ul>
<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">c) For purification the isolated yCD gene:</h3></li>
  
  
<ul>


<li>Once you have run your gel, move it to an open UV box. Remove it from any gel tray as plastic will block most of the UV
</li>
    <li>With a clean, sterile razor blade, slice the desired DNA fragment from the gel.
</li>
    <li>Place the gel in a labeled microfuge tube
</li>
    <li>Using a scale, weigh the tube with the gel fragment after zeroing the scale with an empty tube.
</li>
    <li>Add 200 μL Binding Buffer (NTI) per 100 mg gel.</li>
    <li>Incubate the mixture at 50ºC for 5-10 minutes or until the gel has completely melted.
</li>
    <li>Place a PCR clean-up Gel column in a provided 2 mL collection tube</li>
    <li>Apply 700 μl of the DNA/agarose solution to the PCR clean-up Gel column, and centrifuge at 11,000 x g for 30s at room temperature</li>
    <li>Discard liquid and place the PCR clean-up Gel column back into the same collection tube. For volumes greater than 700 µL, load the column and centrifuge successively, 700 μL at a time. Each PCR clean-up Gel column has a total capacity of 25 μg DNA. If the expected yield is larger, divide the sample into the appropriate number of columns.</li>
    <li>Add 700 μL of Buffer NT3 into the PCR clean-up Gel column. Centrifuge at 11,000 x g for 30s at room temperature to wash the column. Discard the flow-through and re-use the collection tube.</li>
    <li>Repeat step with another 700μL of Buffer NT3.</li>
    <li>Discard liquid and centrifuge the empty PCR clean-up Gel column for 1 minute at 11,000 x g to dry the column matrix. Do not skip this step, it is critical for the removal of ethanol from the PCR clean-up Gel column.</li>
    <li>Place a PCR clean-up Gel column into a clean 1.5 mL eppendorf tube. Add 15-30 μL (depending on desired concentration of final product) of Elution Buffer (10 mM Tris-HCl, pH 8.5) directly onto the column matrix and incubate at room temperature for 1 minute. Centrifuge for 1 minute at 11,000 x g to elute DNA. This represents approximately 70% of bound DNA. An optional second elution will yield any residual DNA, though at a lower concentration.</li>   
   </ul></ul>

   <div style="color:gray;text-align: left;margin: 20px 0 20px 0;">Source: https://www.addgene.org/protocols/gel-electrophoresis/, https://www.protocols.io/viewdna-purification-from-an-agarose-gel-protocol-for-ewov18j32gr2/v1?step=5
</div>

<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">4. Performing error-prone PCR</h3></li>
  <li><h3 style="margin: 0 !important;">Materials and Equipment:</h3></li> 
    <ul>
        <li>100 mM Tris-Cl buffer, pH 8.3
        <li>2 M KCl
        <li>200 mM MgCl2
        <li>25 mM MnCl2
        <li>25 mM, dCTP, pH 7
        <li>25 mM, dTTP, pH 7
        <li>5 mM, dATP, pH 7
        <li>5 mM, dGTP, pH 7
        <li>100 μM 5’ PCR primer
        <li>100 μM 3’ PCR primer
        <li>200 pg/μL DNA template
        <li>5 U/μL Taq DNA polymerase
        <li>100 μL PCR tubes
        <li>Thermal cycler
    </ul>
  <li><h3 style="margin: 0 !important;">Procedure:</h3></li>
    <ul>
        <li>Make a reaction mixture in a 100 μL PCR tube on ice with:
        <li>51 μL water, 10 μL Tris-Cl, 2.5 μL KCl, 3.5 μL MgCl2, 2 μL MnCl2 (Add just before thermal cycling starts), 4 μL (each of dCTP, dTTP, dATP and dGTP), 2 μL each of the 5’ primer and the 3’ primer, 10 μL template DNA and 1 μL Taq DNA polymerase (Add when reaction reaches first annealing step). 
        <li>The tube is then placed in a thermal cycler and enough PCR cycles are performed to achieve 10 doublings with:
        <li>94 °C for 1 minute (for denaturation), 
        <li>60 °C for 1 minute (for annealing) and 
        <li>72 °C for 3 minutes (for extension). 
        <li>The tube is then run on an ethidium bromide-stained agarose gel to confirm the amount of the DNA product.
    </ul>
    </ul> 
    <div style="color:gray;text-align: left;margin: 20px 0 20px 0;">Sources:
• iDEC Resources Wiki<br>
• Copp et al., Directed Evolution Library Creation – Methods and Protocols,
2nd Edition (2014), Springer Protocols<br>
• Wilson, D. S., & Keefe, A. D. (2001), Random Mutagenesis by PCR, Current
Protocols in Molecular Biology. doi:10.1002/0471142727.mb0803s51<br>

</div>  
<h2 style="margin: 0 !important;">5. Double digestion of plasmid backbone</h2>

<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">Materials:</h3></li>
  <ul>
    <li>10-20 µL of plasmid (pET15b) (200-300 ng/μL, total amount 3-5 µg)
    <li>10 µL of 10X Restriction buffer
    <li>10 µL of 10X Bovine Serum Albumin (BSA, final concentration is usually 100 µg/mL)
    <li>1.5-2 µL NdeI (Restriction Enzyme 1) (10-20 units/µL)
    <li>1.5-2 µL BlpI (Restriction Enzyme 2)
    <li>Make up to 70 or 100µL total volume with nuclease free water
  </ul>
  <li><h3 style="margin: 0 !important;">Procedure:</h3></li>
  <ul>
    <li>Add the reagents above in a sterile 1.5 mL Eppendorf, first add the TE or water, then the plasmid pET15b, then the restriction buffer and BSA, and mix thoroughly. Finally, add the restriction enzymes. 
    <li>Incubate the reaction at the right temperature (37 °C) and start the timer. Run the reaction for 40-60 minutes. 
    <li>Run the digest on an agarose gel and inspect the results for a single band the size of your insert.
  </ul>
</ul>

<div style="color:gray;text-align: left;margin: 20px 0 20px 0;">Source: https://www.sigmaaldrich.com/IN/en/technical-documents/technical-article/genomics/cloning-and-expression/restriction-enzyme-cloning-manual-cloning 
</div>

<h2 style="margin: 0 !important;">6. Ligation of the double digested insert and plasmid</h2>

<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">Materials:</h3></li>
  <ul>
    <li>Amplified yCD gene (25-50 ng/μL)
    <li>Plasmid backbone (75 ng/μL)
    <li>Ligation buffer (10X)
    <li>T7 DNA Ligase
    <li>Nuclease free water
  </ul>
    <li><h3 style="margin: 0 !important;">Procedure:</h3></li>
    <ul>
    <li>Mix 1 μL of  the yCD gene obtained from PCR, 1 μL of the plasmid backbone, 1 μL of 10X ligation buffer, 1 μL of T7 DNA ligase (3 units), and nuclease-free water to 10 μL.
    <li>Incubate the ligation mixture for 4 hours at 14℃.
    <li>Store the ligation mixture for transformation.
    </ul>
</ul>

<div style="color:gray;text-align: left;margin: 20px 0 20px 0;">Source: https://static.igem.org/mediawiki/2018/a/a4/T--Tuebingen--RestrictionLigation.pdf</div>

<h2 style="margin: 0 !important;">7. Transforming yCD into E. coli</h2>

<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">Materials:</h3></li>
  <ul>
    <li>LB agar
    <li>SOC medium 
    <li>Appropriate selection antibiotic
    <li>IPTG (Isopropyl-β-D-thiogalactoside
    <li>X-gal, S-Gal
    <li>Ferric Ammonium citrate
  </ul>
  <li><h3 style="margin: 0 !important;">Procedure:</h3></li>
  <ul>
    <li>Take competent E.coli cells from –80℃ freezer. 
    <li>Turn on the water bath to 42℃. 
    <li>Put competent cells in a 1.5 ml eppendorf. 
    <li>For transforming a DNA construct, use 50 ul of competent cells. 
    <li>Add 50 ng of circular DNA into E.coli cells. Incubate on ice for 10 minutes to thaw competent cells. 
    <li>Put tubes with DNA and E.coli into the water bath at 42℃ for 45 seconds. 
    <li>Put tubes back on ice for 2 minutes to reduce damage to the E.coli cells. 
    <li>Add 1 ml of LB (with no antibiotic added). Incubate tubes for 1 hour at 37℃.
    <li>Spread about 100 ul of the resulting culture on LB plates (with appropriate antibiotic added – usually Ampicillin or Kanamycin.)
    <li>Grow the plates overnight at 37℃. 
    <li>Pick transformed colonies 12-16 hours later. 
  </ul>
</ul>

<div style="color:gray;text-align: left;margin: 20px 0 20px 0;">Source: https://web.stanford.edu/~teruel1/Protocols/pdf/Transformation%20Protocol%20Using%20Heat%20Shock.pdf</div>

<h2 style="margin: 0 !important;">8. Extracting cytosine deaminase from E.coli (protein purification with his tag</h2>

<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">Materials and Equipment:</h3></li>
  <ul>
    <li>Ni-NTA superflow (QIAGEN)
    <li>Tris base
    <li>Urea
    <li>IPTG
    <li>NaH2PO4·H2O
    <li>NaOH
    <li>HCl
    <li>LB/Amp media
    <li>Buffer B (pH 8, 1 L): 100 mM NaH2PO4, 10 mM Tris·HCl, 8 M urea 480.5 g Adjust pH to 8.0 using NaOH
    <li>Buffer C (pH 6.3, 1 L): 100 mM NaH2PO4, 10 mM Tris·HCl, 8 M urea
    <li>Buffer D (pH 5.9, 1 L): 100 mM NaH2PO4, 10 mM Tris·HCl, 8 M urea
    <li>Buffer E (pH 4.5, 1L): 100 mM NaH2PO4, 10 mM Tris·HCl, 8 M urea
    <li>Centrifuge
    <li>Sonicator
    <li tyle="list-style-type:none;">Note: Due to the dissociation of urea, the pH of Buffers B, C, D, and E should be adjusted immediately prior to use. Do not autoclave.
  </ul>
  <li><h3 style="margin: 0 !important;">Procedure:</h3></li>
  <li><h3 style="margin: 0 !important;">a) Induction of recombinant proteins</h3></li>
  <ul>
    <li>After picking the transformed colonies, grow 5-10 ml culture to a saturated stage. The next day, inoculate this starter culture in 2 to 4 L of LB/Amp media using 1:50 or 1:100 dilution of saturated culture.
    <li>Grow the culture till it reaches OD= 0.4 to 0.6. Add IPTG to its final concentration of 0.6 M and induce 6x His-tagged protein production for 4 hours.
    <li>Harvest cells using a centrifuge at 5,000 rpm for 20 min. Store cell pellets at -80°C.
  </ul>
  <li><h3 style="margin: 0 !important;">b) Preparation of cleared E. coli lysates under denaturing conditions</h3></li>
  <ul>
    <li>Thaw the cell pellet at room temperature (RT) and resuspend in buffer B at 2 ml per gram wet weight.
    <li>onicate cells in a cold room with setting: Amplitude 30%, 3 min, 15 sec on, 15 sec off. Try 3 min cycles at least twice
    <li>Centrifuge lysate at 10,000X at 8-12 °C to pellet cellular debris
    <li>Collect the supernatant. Save 20 μl as input
  </ul>
  <li><h3 style="margin: 0 !important;">c) Batch purification of 6x His-tagged proteins from E. coli under denaturing conditions</h3></li>
  <ul>
    <li>Add 1 ml of the pre-washed 50% Ni-NTA slurry to 4 ml lysate and mix gently by rotating for 60 min at RT.
    <li>Load lysate-resin mixture carefully into an empty column with the bottom cap still attached.
    <li>Remove the bottom cap and collect the flow through.
    <li>Collect flow through (20 μl) for SDS-PAGE analysis.
    <li>Wash twice with 4 ml buffer C
    <li>Keep wash fractions (20 μl) for SDS-PAGE analysis
    <li>Elute the recombinant protein 4 times with 0.5 ml buffer D, followed by 4 times with 0.5 ml buffer E. 
    <li>Collect fractions and analyze by SDS-PAGE
  </ul>
</ul>

<div style="color:gray;text-align: left;margin: 20px 0 20px 0;">Source: https://bio-protocol.org/exchange/protocoldetail?id=8&type=1</div>

<h2 style="margin: 0 !important;">9. Assaying for enzymatic activity of cytosine deaminase in pH stable buffer</h2>

<p>Note: The conversion of cytosine to uracil by yCD will be measured spectrophotometrically by monitoring change in absorbance at 286 nm, while the conversion of 5FC to 5FU will be monitored at 238 nm. </p>

<ul style="list-style-type:none;">
  <li><h3 style="margin: 0 !important;">Materials and Equipment:</h3></li>
  <ul>
    <li>Buffers at pH 5, 6, 7
    <li>Tris-Cl
    <li>Nanodrop spectrophotometer
  </ul>
  <li><h3 style="margin: 0 !important;">Procedure:</h3></li>
  <ul>
    <li>Dilute the protein to 2µM in 50mM Tris-Cl at fixed pH values of 5,6 and 7 respectively.
    <li>Mix at 1:1 ratio with a range of nine cytosine deaminase concentrations from 0.2–1mM in the same buffer. 
    <li>Measure the absorbance using the nanodrop spectrophotometer at 238 nm every few seconds until baseline is reached and take the first reading after mixing.
    <li>Measurements will be taken in quadruplicate and averaged to reduce error. Initial velocity is calculated as a function of the initial slope by curve-fitting the resulting plot, taking the derivative and extrapolating back to time zero.
    <li>Km and kcat values of wild-type yCD and mutant constructs are determined from a double reciprocal Lineweaver-Burk plot of the resulting data and the catalytic efficiency kcat/Km is calculated from these values.
  </ul>
</ul>

<div style="color:gray;text-align: left;margin: 20px 0 20px 0;">Source: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2330253/</div>
